lib/classes/logged_browser.rb

```diff
@@ -8,6 +8,6 @@ def call
   end
 
   def browser
-    @browser ||= Watir::Browser.new
+    @browser ||= Watir::Browser.new(:chrome)
```

Comment:
On Win 10 with older FF it just wouldn't work. FF usually requires more tinkering and JS fired events rather then the native watir stuff. Maybe I should play with the watir part instead of making this edit but making the browser select trough the mighty constants.rb would be trivial enough.